app/com/lynxanalytics/biggraph/controllers/Operation.scala

```diff
@@ -534,10 +531,11 @@ abstract class TableOutputOperation(context: Operation.Context) extends SmartOpe
   override def apply(): Unit = ???
 }
 
-abstract class ImportOperation(context: Operation.Context) extends TableOutputOperation(context) {
-  import MetaGraphManager.StringAsUUID
-  protected def tableFromGuid(guid: String): Table = manager.table(guid.asUUID)
+case class ImportResult(guid: String, parameterSettings: String)
```

Comment:
This isn't necessarily a guid anymore. Can we rename? Or that would screw up backward compatibilty? If not possible, add a comment here.

Reply:
> This isn't necessarily a guid anymore. Can we rename? Or that would screw up backward compatibilty? If not possible, add a comment here.

Renamed to `key`. (It's the key that identifies the imported data somehow.)

Reply:
> Renamed to `key`. (It's the key that identifies the imported data somehow.)

Sorry, I named it back. The name is used in the Python API too and I don't want trouble.